avocado/utils/cpu.py

```diff
@@ -195,3 +197,59 @@ def offline(cpu):
     if _get_cpu_status(cpu):
         return 1
     return 0
+
+
+def get_cpuidle_state():
+    """
+    Get current cpu idle values
+
+    :return: Dict of cpuidle states values for all cpus
+    :rtype: Dict of dicts
+    """
+    cpus_list = cpu_online_list()
```

Comment:
out of curiosity, why do you only inspect "online" cpus? It's perfectly valid to enable/disable states on disabled cpus (I just tried that). Would you have anything against simple:

```
for cpu in glob.glob("/sys/devices/system/cpu/cpu*:
        ...
```

Reply:
Sure, make sense, will try out in my env and will make that change....